cmd/run/flags.go

```diff
@@ -182,4 +182,13 @@ func bindRunFlags(command *cobra.Command) {
 	flags.Uint32("listObjects-max-results", defaultConfig.ListObjectsMaxResults, "the maximum results to return in ListObjects responses")
 	util.MustBindPFlag("listObjectsMaxResults", flags.Lookup("listObjects-max-results"))
 	util.MustBindEnv("listObjectsMaxResults", "OPENFGA_LIST_OBJECTS_MAX_RESULTS", "OPENFGA_LISTOBJECTSMAXRESULTS")
+
+	flags.Bool("allow-writing-deprecated-models", defaultConfig.AllowWritingDeprecatedModels, "allow writing of deprecated schemas")
+	util.MustBindPFlag("allowWritingDeprecatedModels", flags.Lookup("allow-writing-deprecated-models"))
+	util.MustBindEnv("allowWritingDeprecatedModels", "OPENFGA_ALLOW_WRITING_DEPRECATED_MODELS")
```

Comment:
in config.schema.json you defined `OPENFGA_ALLOW_WRITING_1_0_MODELS` 🤔

Reply:
change it back to OPENFGA_ALLOW_WRITING_1_0_MODELS